common/components/atomic.hpp.inc

```diff
@@ -61,7 +62,7 @@ __forceinline__ __device__ ResultType reinterpret(ValueType val)
     struct atomic_helper<ValueType,                                          \
                          gko::xstd::enable_if_t<(sizeof(ValueType) ==        \
                                                  sizeof(CONVERTER_TYPE))>> { \
-        __forceinline__ __device__ static void atomic_add(                   \
+        __forceinline__ __device__ static ValueType atomic_add(              \
             ValueType *__restrict__ addr, ValueType val)                     \
         {                                                                    \
             CONVERTER_TYPE *address_as_ull =                                 \
```

Comment:
Could you also change this variable name to `address_as_converter`?
I forgot to change it before

Reply:
done